### PR TITLE
Docs: Alerting - On call integration

### PR DESCRIPTION
**Pull request by brendamuir**
- adds cloud release note
- adds oncall integration task docs

### CODE REVIEW

File: docs/sources/alerting/monitor/_index.md [hunk 1]
```diff
@@ -22,6 +22,6 @@ Monitor your alerting metrics to ensure you identify potential issues before the
 [Meta monitoring][meta-monitoring]
 
 {{% docs/reference %}}
-[meta-monitoring]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/monitor/meta-monitoring"
+[meta-monitoring]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/monitor/meta-monitoring/"
```
Comment (jdbaldry) [inline docs/sources/alerting/monitor/_index.md:25]:
```suggestion
[meta-monitoring]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/monitor/meta-monitoring"
```

This argument is passed to the `relref` function and shouldn't have a trailing slash (https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-has-trailing-slash).

**Review by konrad147 — APPROVED** [review]
LGTM!

File: docs/sources/alerting/monitor/_index.md [hunk 2]
```diff
@@ -22,6 +22,6 @@ Monitor your alerting metrics to ensure you identify potential issues before the
...
 [meta-monitoring]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/meta-monitoring"
```
Comment (jdbaldry) [inline docs/sources/alerting/monitor/_index.md:26]: I think this link is out of date.

```suggestion
[meta-monitoring]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/monitor/meta-monitoring"
```

File: docs/sources/alerting/alerting-rules/manage-contact-points/configure-oncall.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+9. To view your integration in the Grafana OnCall application and set up routes and escalation chains, click the Link next to the integration on the Contact points list view page in the **Type** column.
+
+    This redirects you to the Grafana OnCall integration page in the Grafana OnCall application.  From there, you can add [routes and escalation chains][escalation-chain].
+
+{{% docs/reference %}}
+[create-notification-policy]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>alerting/alerting-rules/create-notification-policy/"
+[create-notification-policy]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/alerting-rules/create-notification-policy/"
+
+[oncall-integration]: "/docs/oncall -> /docs/oncall/latest/integrations/grafana-alerting/"
+[oncall-integration]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/oncall/integrations/grafana-alerting/"
```
Comment (jdbaldry) [inline docs/sources/alerting/alerting-rules/manage-contact-points/configure-oncall.md:67]: Currently the link `[oncall-integration]: "/docs/oncall -> /docs/oncall/latest/integrations/grafana-alerting/"` is saying "If this page is published under the path `/docs/oncall` then the link destination should be `/docs/oncall/latest/integrations/grafana-alerting`".

What you want to say is "Wherever this page is published, I want the destination to be `/docs/oncall/latest/integrations/grafana-alerting`"

That is what I have in my suggestion.
```suggestion
[oncall-integration]: "/ -> /docs/oncall/latest/integrations/grafana-alerting"
```

File: docs/sources/alerting/alerting-rules/manage-contact-points/configure-oncall.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+[escalation-chain] "/docs/oncall -> /docs/oncall/latest/escalation-chains-and-routes/"
+[oncall-integration]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/oncall/escalation-chains-and-routes/"
```
Comment (jdbaldry) [inline docs/sources/alerting/alerting-rules/manage-contact-points/configure-oncall.md:70]:
```suggestion
[escalation-chain] "/ -> /docs/oncall/latest/escalation-chains-and-routes"
```

**Review by jdbaldry — CHANGES_REQUESTED** [review]
Please see suggestions.

You also need to run `prettier` on the modified files to get CI to pass.

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 5]
```diff
@@ -29,3 +29,14 @@ The Tempo data source now supports query-type template variables. With this upda
...
 {{< video-embed src="/media/docs/tempo/screen-recording-grafana-10.2-tempo-query-type-template-variables.mp4" >}}
+
+## Grafana OnCall integration for Alerting
+
+<!-- Brenda Muir -->
+<!-- OSS, Enterprise -->
+
+_Generally available in Grafana Cloud_
+
+Use the Grafana Alerting - Grafana OnCall integration to effortlessly connect alerts generated by Grafana Alerting with Grafana OnCall, where you can then route them according to defined escalation chains and schedules.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:40]: I'd think about breaking up this sentence

**Review by imatwawana — APPROVED** [review]
Left one comment, but otherwise, the What's new is good!

**Review by chri2547 — APPROVED** [review]

**Review by jdbaldry — APPROVED** [review]

**Pull request merged**